configure.in

```diff
@@ -749,9 +749,10 @@ WX_ARG_FEATURE(threads,     [  --enable-threads        use threads], wxUSE_THREA
 if test "$wxUSE_MSW" = 1 ; then
 WX_ARG_DISABLE(dbghelp,     [  --enable-dbghelp        use dbghelp.dll API (Win32 only)], wxUSE_DBGHELP)
 WX_ARG_ENABLE(iniconf,      [  --enable-iniconf        use wxIniConfig (Win32 only)], wxUSE_INICONF)
-WX_ARG_FEATURE(regkey,      [  --enable-regkey         use wxRegKey class (Win32 only)], wxUSE_REGKEY)
 fi
 
+WX_ARG_FEATURE(regkey,      [  --enable-regkey         use wxRegKey class (Win32 only)], wxUSE_REGKEY)
```

Comment:
It looks like either regkey check should be placed after "Platform specific tests" section (`USE_WIN32` definition) or this section should be moved before compile options. @vadz what do you think?